### PR TITLE
clean meta_woz

### PR DESCRIPTION
**Pull request by zaidalyafeai**

### CODE REVIEW

File: promptsource/templates/meta_woz/dialogues/templates.yaml [hunk 1]
```diff
@@ -63,10 +69,12 @@ templates:
       \ assistant\"][loop.index % 2]}}: {{utterance}}\n{% endfor %}\n|||\n{{domain.replace('_',\
       \ ' ') | lower }}"
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - BLEU
```
Comment (VictorSanh) [inline promptsource/templates/meta_woz/dialogues/templates.yaml:75]: why isn't it accuracy? there are 47 choices

File: promptsource/templates/meta_woz/dialogues/templates.yaml [hunk 2]
```diff
@@ -76,20 +84,24 @@ templates:
       ][loop.index % 2]}}: {{utterance}}\n{% endfor %}\n|||\n{{domain.replace('_',\
       \ ' ') | lower }}"
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
```
Comment (VictorSanh) [inline promptsource/templates/meta_woz/dialogues/templates.yaml:90]: why isn't it accuracy?

File: promptsource/templates/meta_woz/dialogues/templates.yaml [hunk 3]
```diff
@@ -9,10 +9,12 @@ templates:
       \ 2]}}: {{utterance}}\n{% endfor %}\n|||\n{{domain.replace('_', ' ') | lower\
       \ }}"
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - BLEU
```
Comment (VictorSanh) [inline promptsource/templates/meta_woz/dialogues/templates.yaml:15]: accuracy?

File: promptsource/templates/meta_woz/dialogues/templates.yaml [hunk 4]
```diff
@@ -76,20 +84,24 @@ templates:
...
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
       original_task: true
-    name: predict_next_stmt_1
+    name: predict_second_statement
     reference: ''
```
Comment (VictorSanh) [inline promptsource/templates/meta_woz/dialogues/templates.yaml:105]: remove this one. overlappign with `predict_random_stop`

**Review by VictorSanh — CHANGES_REQUESTED** [review]
lgtm, a few changes

**Comment by zaidalyafeai** [comment]
Is it a good idea to list that many answer choices? I suggest removing templates that predict the domain.

**Comment by VictorSanh** [comment]
> Is it a good idea to list that many answer choices? I suggest removing templates that predict the domain.

I am not suggesting listing the answer_choices, I am talking about the metric. It's a 47 class classification, so the metric so reflect that

**Comment by zaidalyafeai** [comment]
@VictorSanh changed it to accuracy.

**Comment by VictorSanh** [comment]
great thanks @zaidalyafeai ! merging now

**Pull request merged**